exchanges/orderbook/depth.go

```diff
@@ -784,11 +786,29 @@ func (d *Depth) GetTranches(count int) (ask, bid []Tranche, err error) {
 }
 
 // GetPair returns the pair associated with the depth
-func (d *Depth) GetPair() (currency.Pair, error) {
+func (d *Depth) GetPair() currency.Pair {
```

Comment:
[Clarified recent recommendation as part of go docs](https://go.dev/doc/effective_go#Getters)
I think we'll be chiselling away at this for a while, but I like that it's at least specific
```suggestion
func (d *Depth) Pair() currency.Pair {
```